<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_text_field">
       <property name="title">
        <string>Текстовое поле</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::NoWrap</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_tools">
       <property name="title">
        <string>Инструменты</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QSplitter" name="splitter_1">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="btn_record">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Запись</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>image/icon_record.svg</normaloff>image/icon_record.svg</iconset>
            </property>
            <property name="shortcut">
             <string>Alt+M</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_vocalize">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Озвучивать</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>image/icon_volume.svg</normaloff>image/icon_volume.svg</iconset>
            </property>
            <property name="shortcut">
             <string>Alt+V</string>
            </property>
           </widget>
          </widget>
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="btn_play">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Играть</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>image/icon_play.svg</normaloff>image/icon_play.svg</iconset>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_pause">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Пауза</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>image/icon_pause.svg</normaloff>image/icon_pause.svg</iconset>
            </property>
           </widget>
          </widget>
          <widget class="QSplitter" name="splitter_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="btn_copy">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Копировать</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>image/icon_copy.svg</normaloff>image/icon_copy.svg</iconset>
            </property>
            <property name="shortcut">
             <string>Alt+C</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_clear">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Очистить</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>image/icon_trash.svg</normaloff>image/icon_trash.svg</iconset>
            </property>
            <property name="shortcut">
             <string>Alt+X</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>418</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>файл</string>
    </property>
    <addaction name="action_add_document"/>
    <addaction name="action_open"/>
    <addaction name="action_clock_five"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_tools">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="action_record"/>
    <addaction name="action_vocalize"/>
    <addaction name="action_play"/>
    <addaction name="action_pause"/>
    <addaction name="action_copy"/>
    <addaction name="action_clear"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_tools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_record">
   <property name="icon">
    <iconset>
     <normaloff>image/icon_record.svg</normaloff>image/icon_record.svg</iconset>
   </property>
   <property name="text">
    <string>Записть</string>
   </property>
   <property name="shortcut">
    <string>Alt+M</string>
   </property>
  </action>
  <action name="action_vocalize">
   <property name="icon">
    <iconset>
     <normaloff>image/icon_volume.svg</normaloff>image/icon_volume.svg</iconset>
   </property>
   <property name="text">
    <string>Озвучивать</string>
   </property>
   <property name="shortcut">
    <string>Alt+V</string>
   </property>
  </action>
  <action name="action_play">
   <property name="icon">
    <iconset>
     <normaloff>image/icon_play.svg</normaloff>image/icon_play.svg</iconset>
   </property>
   <property name="text">
    <string>Играть</string>
   </property>
  </action>
  <action name="action_pause">
   <property name="icon">
    <iconset>
     <normaloff>image/icon_pause.svg</normaloff>image/icon_pause.svg</iconset>
   </property>
   <property name="text">
    <string>Пауза</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="icon">
    <iconset>
     <normaloff>image/icon_copy.svg</normaloff>image/icon_copy.svg</iconset>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="action_clear">
   <property name="icon">
    <iconset>
     <normaloff>image/icon_trash.svg</normaloff>image/icon_trash.svg</iconset>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="action_add_document">
   <property name="icon">
    <iconset>
     <normaloff>image/icon_add_document.svg</normaloff>image/icon_add_document.svg</iconset>
   </property>
   <property name="text">
    <string>Новый...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset>
     <normaloff>image/icon_open.svg</normaloff>image/icon_open.svg</iconset>
   </property>
   <property name="text">
    <string>Открыть...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset>
     <normaloff>image/icon_save.svg</normaloff>image/icon_save.svg</iconset>
   </property>
   <property name="text">
    <string>Сохранить как...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset>
     <normaloff>image/icon_exit_2.svg</normaloff>image/icon_exit_2.svg</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_clock_five">
   <property name="icon">
    <iconset>
     <normaloff>image/icon_clock_five.svg</normaloff>image/icon_clock_five.svg</iconset>
   </property>
   <property name="text">
    <string>Последний текст</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
